index.html

```diff
@@ -116,7 +116,7 @@ <h4>
 									It's built on GNU/Linux, one of the most secure systems in the world. Same software powering super computers.
```

Comment:
*is* powering?

Reply:
I honestly am not a huge fan of the entire sentence. What about: "It's built on GNU/Linux, one of the most secure systems in the world. The kernel and much of the softare is powering the world's supercomputers every day." ?

Reply:
Sorry, but
1. as you said, on wide screens all the block should have three lines (I wouldn’t notice it otherwise)
2. although GNU is not Linux & Linux is not GNU, kernel is software; mentioning both separately seems a little weird to me
3. *every day* sounds nice, but after analysing it you’ll realise it’s obvious (or do they switch to M$W every other day? :P) — just an idea for shortening it

Reply:
How about then: It even powers the world's super computers.